projects/canopy/src/lib/table/table-head-cell/table-head-cell.component.scss

```diff
@@ -1,18 +1,26 @@
 @import '../../../styles/mixins.scss';
 
+$full-size-breakpoints: 'sm', 'md', 'lg' !default;
```

Comment:
does this need to be re-declared throughout the tables sub components?

Reply:
Agreed, it doesn't. Please see the comment I left: https://github.com/Legal-and-General/canopy/pull/146#discussion_r553318520

Just need to know where Canopy peeps would prefer I put it :)